static/scripts/rewards/web3/erc20-permit.ts

```diff
@@ -74,6 +81,13 @@ export function claimErc20PermitHandler(permit: Erc20Permit, provider: JsonRpcPr
 }
 
 export async function checkPermitClaimable(permit: Erc20Permit, signer: ethers.providers.JsonRpcSigner | null, provider: JsonRpcProvider) {
+  const permitHash = await doesPermitHaveTxHash(permit);
+  if (permitHash !== null) {
+    const explorerLink = `<a href="${getExplorerLinkForTx(permit.networkId, permitHash)}" target=_blank >${shortenTxHash(permitHash)}</a>`;
```

Comment:
Oh I understand, its only if it exists in the database. Perhaps we can do an end-to-end test somehow? How did you test locally?

Reply:
1- generate claim through script
2- write it to db manually (as null transaction column)
3-  and test the rest :) 

I assume that (I have to somehow) permits are written to db correctly.

Reply:
I suppose for another task we should replace the claim button with the transaction hash if it has already been claimed or validated.